eval.py

```diff
@@ -58,8 +56,7 @@ def get_opts():
 @torch.no_grad()
 def batched_inference(models, embeddings,
                       rays, N_samples, N_importance, use_disp,
-                      chunk,
-                      white_back):
```

Comment:
`dataset` is not visible inside this function, so `white_back` in the argument is required.

Reply:
so it's unintuitive, but if you look at the code below, you're never actually calling white_back but instead dataset.white_back (I discovered this because my IDE was flagging the white_back parameter as unused)

The scoping here is really unintuitive here but the IDE and running the code suggest that dataset.white_back is resolving properly in the function)